cmd/crc/cmd/config/config.go

```diff
@@ -23,32 +23,22 @@ const (
 	ProxyCAFile          = "proxy-ca-file"
 )
 
-func RegisterSettings() {
+func RegisterSettings(cfg *config.Config) {
 	// Start command settings in config
-	cfg.AddSetting(Bundle, constants.DefaultBundlePath, cfg.ValidateBundle, cfg.SuccessfullyApplied)
-	cfg.AddSetting(CPUs, constants.DefaultCPUs, cfg.ValidateCPUs, cfg.RequiresRestartMsg)
-	cfg.AddSetting(Memory, constants.DefaultMemory, cfg.ValidateMemory, cfg.RequiresRestartMsg)
-	cfg.AddSetting(NameServer, "", cfg.ValidateIPAddress, cfg.SuccessfullyApplied)
-	cfg.AddSetting(PullSecretFile, "", cfg.ValidatePath, cfg.SuccessfullyApplied)
-	cfg.AddSetting(DisableUpdateCheck, false, cfg.ValidateBool, cfg.SuccessfullyApplied)
+	cfg.AddSetting(Bundle, constants.DefaultBundlePath, config.ValidateBundle, config.SuccessfullyApplied)
+	cfg.AddSetting(CPUs, constants.DefaultCPUs, config.ValidateCPUs, config.RequiresRestartMsg)
+	cfg.AddSetting(Memory, constants.DefaultMemory, config.ValidateMemory, config.RequiresRestartMsg)
+	cfg.AddSetting(NameServer, "", config.ValidateIPAddress, config.SuccessfullyApplied)
+	cfg.AddSetting(PullSecretFile, "", config.ValidatePath, config.SuccessfullyApplied)
+	cfg.AddSetting(DisableUpdateCheck, false, config.ValidateBool, config.SuccessfullyApplied)
+	cfg.AddSetting(ExperimentalFeatures, false, config.ValidateBool, config.SuccessfullyApplied)
 	// Proxy Configuration
-	cfg.AddSetting(ExperimentalFeatures, false, cfg.ValidateBool, cfg.SuccessfullyApplied)
-	cfg.AddSetting(HTTPProxy, "", cfg.ValidateURI, cfg.SuccessfullyApplied)
-	cfg.AddSetting(HTTPSProxy, "", cfg.ValidateURI, cfg.SuccessfullyApplied)
-	cfg.AddSetting(NoProxy, "", cfg.ValidateNoProxy, cfg.SuccessfullyApplied)
-	cfg.AddSetting(ProxyCAFile, "", cfg.ValidatePath, cfg.SuccessfullyApplied)
+	cfg.AddSetting(HTTPProxy, "", config.ValidateURI, config.SuccessfullyApplied)
+	cfg.AddSetting(HTTPSProxy, "", config.ValidateURI, config.SuccessfullyApplied)
+	cfg.AddSetting(NoProxy, "", config.ValidateNoProxy, config.SuccessfullyApplied)
+	cfg.AddSetting(ProxyCAFile, "", config.ValidatePath, config.SuccessfullyApplied)
 }
 
-var (
-	configCmd = &cobra.Command{
-		Use:   "config SUBCOMMAND [flags]",
-		Short: "Modify crc configuration",
-		Run: func(cmd *cobra.Command, args []string) {
-			_ = cmd.Help()
-		},
-	}
-)
-
```

Comment:
Given the size of the diffstat for `Use config object instead of package methods`, this change (removing `init()`, adding 8configXxxCmd()`) could have been split in a preparatory commit, reviews of relatively big commits are much easier when the changes are very mechanical (eg one line removed, same line added with just an additional `config *config.Config`)